eth/eventparser/event_parser_test.go

```diff
@@ -212,3 +212,15 @@ func TestEventParser(t *testing.T) {
 		require.Equal(t, "0xa7a7720499b7eB1F1408A8A319284bFD2Db4a427", parsedEvent.RecipientAddress.String())
 	})
 }
+
+func TestPackOperatorPublicKey(t *testing.T) {
+	t.Run("predefined correct rsa pk bytes", func(t *testing.T) {
+
+		rsaPub := []byte{45, 45, 45, 45, 45, 66, 69, 71, 73, 78, 32, 82, 83, 65, 32, 80, 85, 66, 76, 73, 67, 32, 75, 69, 89, 45, 45, 45, 45, 45, 10, 77, 73, 73, 66, 73, 106, 65, 78, 66, 103, 107, 113, 104, 107, 105, 71, 57, 119, 48, 66, 65, 81, 69, 70, 65, 65, 79, 67, 65, 81, 56, 65, 77, 73, 73, 66, 67, 103, 75, 67, 65, 81, 69, 65, 50, 49, 106, 52, 57, 72, 43, 48, 73, 117, 75, 113, 55, 53, 73, 122, 55, 78, 102, 55, 10, 112, 57, 57, 88, 102, 76, 50, 118, 110, 87, 113, 120, 73, 99, 114, 74, 87, 48, 88, 75, 103, 55, 47, 57, 54, 72, 55, 116, 102, 77, 53, 69, 80, 80, 84, 83, 120, 80, 103, 72, 73, 47, 115, 80, 119, 79, 67, 71, 82, 121, 105, 54, 84, 56, 100, 87, 77, 75, 106, 54, 113, 74, 84, 50, 10, 106, 43, 49, 79, 69, 83, 68, 76, 51, 50, 84, 112, 119, 119, 111, 67, 117, 112, 122, 117, 118, 102, 57, 53, 113, 75, 117, 87, 73, 74, 53, 55, 89, 49, 83, 50, 82, 102, 56, 116, 43, 107, 56, 101, 111, 102, 43, 100, 78, 109, 84, 98, 76, 87, 43, 86, 106, 70, 102, 81, 109, 89, 117, 43, 10, 120, 79, 121, 113, 54, 80, 68, 47, 81, 43, 85, 99, 50, 49, 110, 72, 43, 88, 99, 122, 107, 56, 51, 112, 47, 84, 72, 118, 105, 52, 77, 88, 88, 73, 70, 113, 57, 65, 86, 90, 51, 115, 112, 120, 116, 48, 49, 82, 52, 120, 119, 115, 43, 55, 85, 85, 79, 78, 78, 71, 43, 72, 117, 112, 10, 107, 105, 86, 100, 43, 99, 54, 74, 65, 119, 78, 122, 82, 81, 55, 115, 53, 115, 103, 49, 106, 51, 76, 87, 113, 113, 109, 120, 83, 77, 70, 47, 122, 55, 108, 65, 68, 87, 72, 122, 73, 106, 105, 51, 120, 81, 56, 67, 122, 56, 54, 99, 79, 83, 48, 113, 89, 49, 84, 47, 106, 66, 101, 86, 10, 55, 101, 122, 101, 67, 120, 43, 74, 77, 43, 77, 50, 109, 101, 90, 65, 74, 112, 49, 79, 100, 109, 97, 98, 98, 82, 43, 97, 77, 121, 79, 100, 76, 43, 106, 72, 116, 75, 53, 105, 116, 78, 88, 87, 49, 52, 88, 81, 47, 89, 112, 83, 76, 86, 84, 76, 51, 90, 80, 90, 79, 119, 119, 78, 10, 99, 81, 73, 68, 65, 81, 65, 66, 10, 45, 45, 45, 45, 45, 69, 78, 68, 32, 82, 83, 65, 32, 80, 85, 66, 76, 73, 67, 32, 75, 69, 89, 45, 45, 45, 45, 45, 10}
```

Comment:
Perhaps providing data as a hex string and decoding it afterwards would be more readable, but if you disagree then ignore this comment